library/pod-security-policy/allow-privilege-escalation/template.yaml

```diff
@@ -3,12 +3,24 @@ kind: ConstraintTemplate
 metadata:
   name: k8spspallowprivilegeescalationcontainer
   annotations:
-    description: Controls restricting escalation to root privileges.
+    description: >-
+      Controls restricting escalation to root privileges. Corresponds to the
+      `allowPrivilegeEscalation` field in a PodSecurityPolicy. For more
+      information, see
+      https://kubernetes.io/docs/concepts/policy/pod-security-policy/#privilege-escalation
 spec:
   crd:
     spec:
       names:
         kind: K8sPSPAllowPrivilegeEscalationContainer
+      validation:
+        # Schema for the `parameters` field
```

Comment:
This template does not have parameters. Would this comment confuse people; expecting it to have parameters?

Reply:
this applies to privileged policy too

Reply:
removed for both